packages/wallets/src/components/WalletListCardDetails/__tests__/WalletListCardDetails.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { APIProvider } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import WalletListCardDetails from '../WalletListCardDetails';
+
+describe('WalletListCardDetails', () => {
+    it('should render with active demo account details correctly', () => {
+        render(
+            <APIProvider>
+                <WalletListCardDetails isActive isDemo loginid='VRW123456' title='USD' />
+            </APIProvider>
+        );
+        expect(screen.getByText('USD Wallet')).toBeInTheDocument();
+        expect(screen.getByText('Reset balance')).toBeInTheDocument();
+    });
+
+    it('should render with active real account details correctly', () => {
+        render(
+            <APIProvider>
+                <WalletListCardDetails isActive isDemo={false} loginid='VRW123456' title='USD' />
```

Comment:
```suggestion
                <WalletListCardDetails isActive isDemo={false} loginid='CRW123456' title='USD' />
```
Not that it makes a difference. Just caught my eye 😅